### ISSUE TITLE
WA and UA of the training set

### ISSUE THREAD

**Issue by wangdtfyguhjknjbhvgc** [Vincent-ZHQ/CA-MSER#24]
Hi, I wrote the WA and UA results for each epoch of the training set to tensorboard and found that the accuracy fluctuates around 25% without rising. I tried checking for various issues but did not find a cause. Is this reasonable please? Or is the accuracy rate calculated incorrectly? This question has been bothering me for a long time, and I am very much looking forward to getting your reply!

**Comment by wangdtfyguhjknjbhvgc** [comment]
In train_ser.py, def train(); 
train_loader = torch.utils.data.DataLoader(train_dataset,batch_size=params['batch_size'],shuffle=True)
shuffle should be changed to Fasle. When calculating the WA and UA, the labels dont correspond to the data.
